.github/workflows/main.yml

```diff
@@ -0,0 +1,29 @@
+name: Android CI
+
+on: [push, pull_request]
+
+jobs:
+  apk:
+    name: Generate APK
+    runs-on: ubuntu-18.04
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Create empty local.properties
+        uses: finnp/create-file-action@1.0.0
+        env:
+          FILE_NAME: "local.properties"
```

Comment:
Is it necessery? Which task fails without it?

Reply:
https://github.com/bahusoid/orion-viewer/runs/649281860

```
* Where:
Script '/home/runner/work/orion-viewer/orion-viewer/project.gradle' line: 3

* What went wrong:
A problem occurred evaluating script.
> /home/runner/work/orion-viewer/orion-viewer/local.properties (No such file or directory)
```

If you can make it optional - I will remove it.

Reply:
Done

Reply:
Is it possible to include date/time stamp in archive name?

Reply:
Or maybe just upload 4 files as is, because they are already zipper?

Reply:
> Or maybe just upload 4 files as is, because they are already zipped?

Would it work if not specify FILE_NAME?

Reply:
I removed empty file creation task.

> Is it possible to include date/time stamp in archive name?

I guess it is but I don't know how :).  I found how to add unique build number so name is different for each build. I think it's good enough :) 

Last Log: https://github.com/bahusoid/orion-viewer/runs/656937121
Artifacts: https://github.com/bahusoid/orion-viewer/suites/665189348/artifacts/5909178

Reply:
I've adjusted name to make it more clear. Now it looks like OrionViewer-CI-build22.zip